### DISCUSSION TITLE
Width of contents

### DISCUSSION THREAD

**Discussion by cupcakearmy** [wojtekmaj/react-pdf#1571] [Q&A]
How do I get a container to have the width of the contents?

**Comment by wojtekmaj** [comment]
For generic HTML/CSS questions, please head to StackOverflow.

If you for some reason explicitly want to use currently rendered page width for something, onRenderSuccess callback on Page component passes page object that you can access width, height, originalWidth, originalHeight params in.

**Reply by cupcakearmy** [reply]
Thanks for the answer, I realise my question is badly formulated, sorry.

Basically the issue is that by default everything is a flex box, so I wanted to ask if there is a way to limit the width of the containing element to the width of the childs. In css we have `fit-content` width.

My solution rn is to just encase it into another flex parent container and let it shrink.

**Reply by wojtekmaj** [reply]
You probably don't want to just "let PDFs shrink".

This might be helpful: https://github.com/wojtekmaj/react-pdf/issues/545
